src/Merger.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TomasVotruba\PhpFwTrends;
+
+use TomasVotruba\PhpFwTrends\ValueObject\VendorData;
+use Webmozart\Assert\Assert;
+
+final class Merger
```

Comment:
This class needs tests.

"Merger" could be related to anything between 2 items. More precise name would help to understand too.